daemon/config.go

```diff
@@ -70,6 +70,8 @@ type CommonConfig struct {
 	Root                 string              `json:"graph,omitempty"`
 	SocketGroup          string              `json:"group,omitempty"`
 	TrustKeyPath         string              `json:"-"`
+	CorsHeaders          string              `json:"api-cors-headers,omitempty"`
+	EnableCors           bool                `json:"api-enable-cors,omitempty"`
```

Comment:
This field can also stay in were it was before, we only need `CorsHeaders` in both platforms.